Refactor outfit skillchips

coiax opened the pull request:
- Skillchips now are on /datum/outfit, rather than /datum/outfit/job,
  since we may want other roles to start with skillchips in the future.
- Skillchips are now a list of types to install, rather than a single
  one.
- Traits for the deathsquad are now listed just below the deathsquad
  initial antag datum definition.

coiax committed:
🏭 Refactor outfit skillchips

- Skillchips now are on /datum/outfit, rather than /datum/outfit/job,
  since we may want other roles to start with skillchips in the future.
- Skillchips are now a list of types to install, rather than a single
  one.
- Traits for the deathsquad are now listed just below the deathsquad
  initial antag datum definition.

Fikou commented on code/modules/jobs/job_types/janitor.dm:
```diff
@@ -24,6 +24,7 @@
 	ears = /obj/item/radio/headset/headset_srv
 	uniform = /obj/item/clothing/under/rank/civilian/janitor
 	backpack_contents = list(/obj/item/modular_computer/tablet/preset/advanced=1)
+	skillchips = list(/obj/item/skillchip/job/engineer, /obj/item/skillchip/light_remover)
```
unmentioned balance change thats also really powerful 🤔

cacogen replied:
One is removing light tubes without gloves while the other is door wires? Is that right?

Jared-Fogle replied:
yes

coiax replied:
Oh shit, I didn't mean to leave that in, that was for testing :sweat:

cacogen replied:
Light remover makes sense. I don't get the engineer one. The janitor should have a chipset that makes them clean faster.

coiax replied:
Feel free to make a PR adding the light remover skillchip to the janitor after this one is merged, but this is just supposed to be pure refactoring.

coiax committed:
Remove accidental debug testing code

It was not intended for janitors to start with engineering skillchips,
and a really weak lightbulb removing skillchip.

coiax committed:
Fix indentation of comment
